fr/administration/centreon-ha/installation-2-nodes.md

```diff
@@ -662,6 +662,8 @@ pcs quorum device add model net \
 
 Cette commande ne doit être lancée que sur un des deux nœuds centraux :
 
+hoho
```

Comment:
@omercier funny but I don't understand what it means :)

Reply:
OMG!